frameworks/Java/act/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.sonatype.oss</groupId>
+        <artifactId>oss-parent</artifactId>
+        <version>7</version>
+    </parent>
+
+    <groupId>com.techempower</groupId>
+    <artifactId>actframework</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <name>ActFramework sample project</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <act.version>0.3.1-SNAPSHOT</act.version>
+        <act-morphia.version>0.4.1-SNAPSHOT</act-morphia.version>
+        <act-ebean.version>0.3.1-SNAPSHOT</act-ebean.version>
```

Comment:
Are those available at Maven central ?
I cannot find them.
And I don't see custom repositories in this file.